jwst/pipeline/tests/test_calwebb_spec2.py

```diff
@@ -0,0 +1,63 @@
+"""Test calwebb_spec2"""
+
+from contextlib import contextmanager
+from os import path
+import pytest
+
+from ...associations.asn_from_list import asn_from_list
+from ...associations.lib.rules_level2_base import DMSLevel2bBase
+from ...datamodels import open as dm_open
+from ..calwebb_spec2 import Spec2Pipeline
+
+
+def abspath(filepath):
+    """Get the absolute file path"""
+    return path.abspath(path.expanduser(path.expandvars(filepath)))
+
+
+DATAPATH = abspath(
+    '$DEVDIR/testdata/jwst_data/dev/build7.1/spec2_test'
+)
+EXPFILE = 'jw00035001001_01101_00001_mirimage_rate.fits'
```

Comment:
That's appropriate. The LRS spectra fall on the imager detector, so the file name will have the field "mirimage", because that's the name of the detector (like nrc1a, nrcblong, guider1). Coronagraphic data also fall on the imager detector. So the down side to this is that you can't tell just by glancing at the file name whether you've got an LRS fixed-slit, LRS slitless, coronagraphic, or imaging mode exposure. You have to look inside at exp_type.